RecoDataProducts/inc/BkgCluster.hh

```diff
@@ -14,14 +14,15 @@ namespace mu2e
    struct BkgCluster 
    {
        //Default hit count chosen for compuational efficiency
-       BkgCluster() : _time(0.0), _flag(BkgClusterFlag::update)  {_hits.reserve(16);}   
-       BkgCluster(XYZVec const& pos, float time) : _pos(pos), _time(time),_flag(BkgClusterFlag::update) {_hits.reserve(16);}
+       BkgCluster()                              : _pos(0.0,0.0,0.0), _time(0.0),  _hits(), _flag(BkgClusterFlag::update)  {_hits.reserve(16);}   
```

Comment:
The default XYZ constructor should give (0,0,0), why override that?

Reply:
I didn't look at the default constructor so I put explicitly (0,0,0), but I have changed the code to use the default constructor.